src/Components/Panels/Inventory/LinkSummary.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  faLongArrowAltRight,
+  faLongArrowAltLeft,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React, { forwardRef } from 'react';
+
+import { IBaseObject } from '../../../Document/Obstacles/BaseObject';
+import { ITeleporter } from '../../../Document/Obstacles/Teleporter';
+import { TeleporterSide } from '../../../Document/Obstacles/TeleporterLink';
+
+import styles from './ObstacleSummary.module.scss';
+
+interface Props {
+  object: IBaseObject;
+}
+
+const LinkSummary = forwardRef<HTMLDivElement, Props>(({ object }: Props) => {
```

Comment:
No need for the `forwardRef` function call. Just make the `LinkSummary` an arrow function, not a component with a forwarded ref.

```suggestion
const LinkSummary = ({ object }: Props) => {
```